redi-connect-front/src/components/atoms/MediaIcons.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../styles/variables.scss';
+
+.media {
+  &-icons {
```

Comment:
Can we use a single convention for these "subclasses"? I'm thinking we can re-use [this clean approach](https://github.com/talent-connect/connect/pull/56/files#diff-a5c4952b4cb1e9960589f357adff8f1e) I saw in the last PR I reviewed.

Reply:
The root class here simply named "media" isn't very descriptive. Consider the following changes:

- `.media` => `.media-icons`

Furthermore,
- `icons` => `wrapper`

Reply:
Good idea, changed as suggested